### ISSUE TITLE
sentinl 5 and kibana 5: index array is not used to query elasticsearch

### ISSUE THREAD

**Issue by fbaligand** [sirensolutions/sentinl#128]
Hi,

Using sentinl 5.2.2 with kibana 5.2.2 and sentinl 5.3.2 with kibana 5.3.2, I reproduce the following issue:
- I set the following input :
```
{
  "search": {
    "request": {
      "index": [
        "my-index"
      ],
      "body": {
        "query": {
          "match_all": {}
        }
      }
    }
  }
}
```

- And when query is executed by sentinl, it calls `/_search` instead of `my-index/_search`.
- I saw clearly it because I use a TCP/IP monitor to watch executed queries by kibana/sentinl.
Thus, the returned payload.hits.total matches cluster total hits, and not `my-index` total hits.

**Comment by fbaligand** [comment]
@SergeyBondarenko 
After looking at code, it seems that some times, "search.request.index" is used, some other times, "search.request.indices" is used, which explains the bug.

**Comment by SergeyBondarenko** [comment]
The fix for the bug is in the master branch. It will be in the next release too.

**Comment by fbaligand** [comment]
Nice news !
Thanks for the fix @SergeyBondarenko !

**Comment by fbaligand** [comment]
I tried the fix in v5.2.2-1 and it works great !
=> thanks @SergeyBondarenko !
